apps/expo-go/ios/Podfile.lock

```diff
@@ -3081,7 +3081,7 @@ SPEC CHECKSUMS:
   Google-Maps-iOS-Utils: cfe6a0239c7ca634b7e001ad059a6707143dc8dc
   GoogleMaps: 80ea184ed6bf44139f383a8b0e248ba3ec1cc8c9
   GoogleUtilities: 26a3abef001b6533cf678d3eb38fd3f614b7872d
-  hermes-engine: fc27a3c4b51c81f907a3e55b33fd71bae214c9f1
+  hermes-engine: 699c9e93f5af5892ce55dabed81ca70680bb356d
```

Comment:
Are you sure it's the correct hash? Can you remove `ios/build` and `ios/Pods` and reinstall pods?

Reply:
Seems to be, I do that whenever I get lock changes